README.md

```diff
@@ -27,52 +27,52 @@ npm i --save @hubspot/calling-extensions-sdk
 
 ### Installation
-1. Clone or fork this repository to download the Demo Call Widget.
-2. Open your terminal, navigate to the root directory of the project, and then switch to the demo widget directory by running one of the following commands.
-- For demo-minimal-js:
+
+Install these demos on your local environment using the following steps, or skip to the next section to run the demos without installation.
+1. Ensure that you have installed [Node.js](https://nodejs.org/en/) on your environment.
+2. Clone, fork, or download the ZIP of this repository.
+3. Open your terminal, navigate to the root directory of the project, and run one of the following commands:
+- For the `demo-minimal-js`:
   ```shell
-  cd demos/demo-minimal-js
+  cd demos/demo-minimal-js && npm i && npm start
   ```
-- For demo-react-ts:
+- For the `demo-react-ts`:
   ```shell
-  cd demos/demo-minimal-js
+  cd demos/demo-react-ts && npm i && npm start
   ```
-3. Follow the instructions in the [demo-minimal-js](https://github.com/HubSpot/calling-extensions-sdk/tree/project-demo-v1/demos/demo-minimal-js) or [demo-react-ts](https://github.com/HubSpot/calling-extensions-sdk/tree/project-demo-v1/demos/demo-react-ts) folder to install the [Node.js](https://nodejs.org/en/) dependencies required for the project using the [npm CLI](https://docs.npmjs.com/cli/v9) and run the app.
+  These will switch to the desired demo directory, install the [Node.js](https://nodejs.org/en/) dependencies required for the project using the [npm CLI](https://docs.npmjs.com/cli/v9), and start the app. Note that the `npm start` command will automatically open a new tab in your browser at https://localhost:9025/, and you may need bypass a "Your connection is not secure" warning in order to access the application.
+
 
 ### Launch the demo call widget from HubSpot
-1. Navigate to a contact or company page within your HubSpot account.
-2. Open the browser's developer console, paste the following code into the console based on which demo widget you want to launch, and press Enter to run it:
-- For demo-minimal-js:
-  ```js
-  localStorage.setItem(
-    "LocalSettings:Calling:CallingExtensions",
-    '{"name": "Demo Widget JS", "url": "https://localhost:9025/"}'
-  );
 
-  // OR
-  localStorage.setItem(
-    "LocalSettings:Calling:installDemoWidget", "local:js"
-  );
-  ```
-- For demo-react-ts:
+1. Navigate to a contact or company page within your HubSpot account.
+2. Open your browser's developer console, and run one of the following:
+- If you've installed the `demo-minimal-js` or the `demo-react-ts`:
   ```js
   localStorage.setItem(
     "LocalSettings:Calling:CallingExtensions",
-    '{"name": "Demo Widget React", "url": "https://localhost:9025/"}'
-  );
-
-  // OR
-  localStorage.setItem(
-    "LocalSettings:Calling:installDemoWidget", "local"
+    '{"name": "Demo Widget Local", "url": "https://localhost:9025/"}'
```

Comment:
Do we need to make any changes to the `DEMO_WIDGET_VALUES`?

Reply:
My reasoning is that since we're offering to run the demos with and without installation that there's already a lot of options. In the case that they've installed locally, this command works for both demos, and we can use `local` and `local:js` internally. Happy to change it back if you prefer.

Reply:
Yeah we do have a lot of options for installing locally. I was just wondering if you had in mind the idea to combine `local` and `local:js` when I saw this change

Reply:
Ah, I see! I can do that. I'll simplify to these:
```ts
export enum DEMO_WIDGET_VALUES {
  APP_MINIMAL_JS = 'app:js',
  APP_REACT_TS = 'app',
  LOCAL = 'local',
}
```

Reply:
Then I'll update this README

Reply:
https://github.com/HubSpot/calling-extensions-sdk/pull/117/files